agr_literature_service/lit_processing/data_ingest/post_reference_to_db.py

```diff
@@ -141,6 +164,7 @@ def insert_mod_corpus_associations(db_session, primaryId, reference_id, mod_to_m
             log.info(primaryId + ": INSERT MOD_CORPUS_ASSOCIATION: for reference_id = " + str(reference_id) + ", mod_id = " + str(mod_id) + ", mod_corpus_sort_source = " + x['modCorpusSortSource'])
         except Exception as e:
             log.info(primaryId + ": INSERT MOD_CORPUS_ASSOCIATION: for reference_id = " + str(reference_id) + ", mod_id = " + str(mod_id) + ", mod_corpus_sort_source = " + x['modCorpusSortSource'] + " " + str(e))
+    return mod_id
```

Comment:
(I mean, I get that it would only ever return the last one, just wondering if it matters)

Reply:
good catch! and good question! @azurebrd 

Since workflow tags are from mod dqm reference files, it should be only one mod in modCorpusAssociation data field. 

If the data is from ABC, it may have multiple mod_ids associated with a paper, so everything should be fine.

Reply:
Cool, thanks !